setup.py

```diff
@@ -22,12 +22,14 @@
     python_requires=">=3.8",
     install_requires=[
         "requests",
+        "attr",
```

Comment:
This is pretty confusing, but the `import attr` used in the model files refers actually to the `attrs` package (`attr` also exists, but that is something else)